tests/unit/metadata/test_single_table.py

```diff
@@ -481,6 +481,58 @@ def test__validate_column_text(self, mock__validate_text, mock__validate_kwargs)
             'phrase', 'text', regex_format='[A-z0-9]', pii=True)
         mock__validate_text.assert_called_once_with('phrase', regex_format='[A-z0-9]', pii=True)
 
+    @patch('sdv.metadata.single_table.SingleTableMetadata._validate_unexpected_kwargs')
+    def test__validate_column_pii_not_true_or_false(self, mock__validate_kwargs):
+        """Test ``_validate_column`` method when ``pii`` is not ``True``or ``False``."""
+        # Setup
+        instance = SingleTableMetadata()
+
+        # Run and Assert
+        error_msg = re.escape(
+            "Parameter 'pii' is set to an invalid attribute (some_text). Expected a value of "
+            'True or False.'
+        )
+        with pytest.raises(InvalidMetadataError, match=error_msg):
+            instance._validate_column('address', 'address', pii='some_text')
+
+    @patch('sdv.metadata.single_table.SingleTableMetadata._validate_unexpected_kwargs')
+    def test__validate_column_pii(self, mock__validate_kwargs):
+        """Test ``_validate_column`` method when ``pii`` is ``True``or ``False``."""
+        # Setup
+        instance = SingleTableMetadata()
+
+        # Run and Assert
+        instance._validate_column('address', 'address', pii=True)
+
+    def test_update_column_sdtype(self):
```

Comment:
Just curious, why did you add this and the test below?

Reply:
because they were missing and I just realized that while looking into the unit tests (I was going to use `update_column` to test the implementation but then I saw that those are missing and that I have to unit test it on the validate).